docs/tableconfig_schema.rst

```diff
@@ -0,0 +1,204 @@
+
+  * Allowed values - Positive integers
+
+* ``schemaName`` - Name of the schema that's been uploaded to the controller
+
+  * Allowed values - String
+
+* Time column - using ``timeColumnName`` and ``timeType``, this must match what's configured in the preceeding schema
+
+  * Allowed values - String, this must match the ``timeFieldSpec`` section in the schema
+
+* Segment assignment strategy - Described more on the page `Customizing Pinot <customizations.html#segment-assignment-strategies>`_
+
+
+.. code-block:: none
+
+    "segmentsConfig": {
+      "retentionTimeUnit": "DAYS",
+      "retentionTimeValue": "5",
+      "segmentPushFrequency": "daily",
+      "segmentPushType": "APPEND",
+      "replication": "3",
+      "replicasPerPartition": "3",
+      "schemaName": "myPinotSchmea",
+      "timeColumnName": "daysSinceEpoch",
+      "timeType": "DAYS",
+      "segmentAssignmentStrategy": "BalanceNumSegmentAssignmentStrategy"
+    },
+
+Table Index Config Section
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``tableIndexConfig`` section has information about how to configure:
+
+* ``invertedIndexColumns`` - Indicates a list of real column names as specified in the schema to create inverted indexes for. More info on indexes can be found on the `Index Techniques <index_techniques.html>`_ page.
+
+  * Allowed values - String; string must match the column name in the corresponding schema
+
+* ``noDictionaryColumns`` - Indicates a list of real column names as specified in the schema. Column names present will **not** have a dictionary created. More info on indexes can be found on the `Index Techniques <index_techniques.html>`_ page.
+
+  * Allowed values - String; string must match the column name in the corresponding schema
+
+* ``sortedColumn`` - Indicates a list of real column names as specified in the schema. Data should be sorted based on the column names provided. More info on indexes can be found on the `Index Techniques <index_techniques.html>`_ page.
+
+  * Allowed values - String; string must match the column name in the corresponding schema
+
+* ``aggregateMetrics`` - Switch for the aggregate metrics feature. This feature will aggregate realtime stream data as it is consumed, where applicable, in order to reduce segment sizes. This feature is only available on REALTIME tables.
+
+  * Allowed values - ``true`` to enable, ``false`` to disable.
+
+* ``segmentPartitionConfig`` - Cofigures the Data Partitioning Strategy. Further documentation on this feather available in the `Data Partitioning Strategies <customizations.html#data-partitioning-strategies>`_ section.
+* ``loadMode`` - indicates how data will be loaded on pinot-server. either ``"MMAP"`` or ``"HEAP"`` can be configured.
+
+  * Allowed values:
+
+    * ``MMAP`` - Configures pinot-server to load data segments to off-heap memory.
+    * ``HEAP`` - Configures pinot-server to load data directly into heap memory.
```

Comment:
Unfortunate misnomer of HEAP. It actually loads segment into direct memory, not heap memory

Reply:
heap -> direct